tests/ai_safety/_canary_stray.py

```
@@ -0,0 +1,3 @@
+STRAY_MINIO = "quay.io/fake-org/minio-stray:v2024.1"
+STRAY_VLLM = "ghcr.io/fake-org/vllm-stray:0.4.0"
+SUPPRESSED = "quay.io/fake-org/should-not-flag:v1"  # noqa: IMG001
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: IMG001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)